subworkflows/local/align_cellrangermulti.nf

```diff
@@ -112,7 +117,12 @@ workflow CELLRANGER_MULTI_ALIGN {
                     lines = [ 'sample_id,probe_barcode_ids,description' ]
                     meta[0].each{ lines = lines + [ "$it.multiplexed_sample_id,$it.probe_barcode_ids,$it.description" ] }
                     frna = file( "${workflow.workDir}/${sample}_frna_samplesheet.csv" )
```

Comment:
I'm not sure this is a good idea. 
 * Does this work on an AWS batch setup? 
 * there could be clashes if multiple workflows run with the same work dir
 * the file could be leftover from previous pipeline runs

Maybe this needs to become its own process that has its own work dir, or at least write to a unique filename.